subsys/bootloader/bl_storage/bl_storage.c

```diff
@@ -4,54 +4,42 @@
  * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
  */
 
-#include "provision.h"
+#include "bl_storage.h"
 #include <string.h>
-#include <stdbool.h>
 #include <errno.h>
 #include <nrf.h>
 #include <assert.h>
 #include <pm_config.h>
 #include <nrfx_nvmc.h>
 
 
-typedef struct {
+struct bl_storage_data {
```

Comment:
Could we move this to the .h file? I will need it there soon, and I don't see why it shouldn't be there.

Reply:
We decided not to move it, it's not needed after all.